[
    {
        "problem_type": "Invalid Configuration",
        "test_name": "test_random_reviewers_no_weight",
        "language": "Python",
        "filename": "mergify_engine/tests/unit/actions/test_request_reviews.py",
        "lineno": 63,
        "error": "MultipleInvalid",
        "test_framework": "pytest",
        "stack_trace": "voluptuous.error.MultipleInvalid: GitHub login contains invalid characters: jd @ data['users']"
    },
    {
        "problem_type": "Invalid Configuration",
        "test_name": "test_github_team_ok",
        "language": "Python",
        "filename": "mergify_engine/tests/unit/rules/test_types.py",
        "lineno": 118,
        "error": "MultipleInvalid",
        "test_framework": "pytest",
        "stack_trace": "voluptuous.error.MultipleInvalid: GitHub organization contains invalid characters: foo"
    }
]